### PR TITLE
Update homepage, main events page and single workshop page to use Bootstrap 5

### PR DESCRIPTION
**Pull request by KimberleyCook**
Even though there are lots of files it isn't that bad I promise.

There are notes about deleting CSS in some of the files that I'm unsure if I can delete just yet as I'm not 100% sure whether deleting it will break another page, so notes have been added as a very last task to delete all the unused CSS (that I don't delete as I go).

Updated homepage (with new dropdown menu)
screencapture-localhost-3000-2020-11-05-17_12_27

Updated single workshop page

screencapture-localhost-3000-workshops-4-2020-11-05-17_13_11

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -6,11 +6,12 @@ gem 'rails', '4.2.11.3'
 gem 'acts-as-taggable-on', '~> 5.0'
 gem 'carrierwave'
 gem 'carrierwave-ftp', require: 'carrierwave/storage/sftp'
-gem 'compass-rails', git: 'https://github.com/Compass/compass-rails'
+gem 'compass-rails'
```
Comment (KimberleyCook) [inline Gemfile:9]: I had to update this to get the new Bootstrap working, let me know @despo if you think it'll be an issue.

Comment (despo) [reply Gemfile:9]: No idea! I thought it had to do with foundation but that does seem to be the case (related commit: 
https://github.com/codebar/planner/commit/7b57dc5fc9d6c7baef5ebb3f58b305d14be9a94b)

We should probably try to push this fix on its own to staging and see if it causes any issues with deployment. If not we should be fine.

Comment (KimberleyCook) [reply Gemfile:9]: Sounds good, I'll do a separate PR for this to test staging 👍

Comment (KimberleyCook) [reply Gemfile:9]: Separate PR here https://github.com/codebar/planner/pull/1421

**Comment by KimberleyCook** [comment]
Screwed up a rebase so closing this PR, and opening a new one!

**Pull request closed without merging**